File: unity_app/Assets/Scripts/Core/Enums/MapEnums.cs
namespace EventPlannerSim.Core
{
    /// <summary>
    /// Represents a geographic area of the city map containing related venues and vendors.
    /// </summary>
    public enum MapZone
    {
        Neighborhood,
        Downtown,
        Uptown,
        Waterfront
    }

    /// <summary>
    /// Represents the type of location on the map.
    /// </summary>
    public enum LocationType
    {
        All,
        Venue,
        Vendor,
        Office,
        MeetingPoint
    }
}

File: original-source/Models/Enums/MapEnums.swift
import Foundation

enum MapZone: String, CaseIterable, Codable, Hashable {
    case neighborhood
    case downtown
    case uptown
    case waterfront
}

enum LocationType: String, CaseIterable, Codable, Hashable {
    case all
    case venue
    case vendor
    case office
    case meetingPoint
}
